lms/services/canvas_api/client.py

```diff
@@ -60,6 +61,13 @@ def __init__(
         self._file_service = file_service
         self._folders_enabled = folders_enabled
 
+        self._pages = CanvasPagesClient(authenticated_client)
```

Comment:
Using composition here for the pages part.

Reply:
Should it be injected like the `FileService`?

Reply:
Yeah, I was going to comment that that would require CanvasPagesClient to be a service but that's true.

We can inject this simple constructions, it building those gets more complicated the client doesn't' need to know about it :+1: